modules/serverless-rn-1-33-0.adoc

```diff
@@ -18,7 +18,10 @@
 * {ServerlessProductName} now uses Knative for Apache Kafka 1.12.
 * The `kn func` CLI plugin now uses `func` 1.14.
 
+* OpenShift Serverless Logic is now generally available (GA). This release includes an overview of OpenShift Serverless Logic, instructions on creating, managing, and deploying workflows, and guidelines for the installation and uninstallation of {ServerlessLogicOperatorName}. Additionally, it includes steps for configuring OpenAPI services and endpoints, and techniques for troubleshooting the services. For more information, see link:https://docs.openshift.com/serverless/latest/about/serverless-logic-overview.html[OpenShift Serverless Logic overview].  
```

Comment:
I'm not familiar enough with serverless docs to know this - do you guys not use xrefs for internal links?

Reply:
Yes, added the xref link now.